README.md

```diff
@@ -1,11 +1,29 @@
 # Stockpile App
 Front end for Stockpile, an app that manages stuff for organizations.
 
+Built with [Ionic 2](https://github.com/driftyco/ionic).
+
 ## Getting Started
 - Clone this repo
-- Make sure you have the lastest version of Ionic and Cordova installed `npm install -g ionic cordova`
+- Make sure you have the latest version of Ionic and Cordova installed `npm install -g ionic cordova`
 - Run `npm install` from the project root
-- Add [Barcode Scanner Plugin](https://ionicframework.com/docs/v2/native/barcode-scanner/): `ionic plugin add phonegap-plugin-barcodescanner`
-- Run `ionic serve` to view the app in your browser
+- Run `ionic serve` to preview the app in your browser
+- You can also run `ionic serve --lab` to preview different platforms side-by-side
 
 **Note:** If your build is slow, update `npm` to 3.x: `npm install -g npm`.
```

Comment:
You're probably right, but doesn't hurt to keep it in there in case someone else wants to run the project (Dr. Tuck for grading or another student during other team's project review)